frontend/middleware.ts

```diff
@@ -0,0 +1,120 @@
+// This is a Vercel middleware that sends HTML containing fake HTML for social
+// media links
+import { next } from '@vercel/edge';
+
+// For Prettier formatting. If you add a language tag before the template
+// literal, it will recognize them as embedded languages and format those
+const identity = (strings: TemplateStringsArray, ...values: unknown[]) =>
+  String.raw({ raw: strings }, ...values);
+const html = identity;
+const gql = identity;
```

Comment:
@bearsyankees Not sure why you changed it but FYI the names `html` and `gql` are important for Prettier to format them